recipes/pyunlocbox/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pyunlocbox" %}
+{% set version = "0.5.1" %}
+{% set sha256 = "50e6e900fe321e6335a6cf71854156075650068bd75980a33f1ae3b83215f5ca" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - scipy
+  run:
+    - python
+    - numpy
+    - scipy
+
+test:
+  imports:
+    - pyunlocbox
+    - pyunlocbox.tests
```

Comment:
No need to import the `tests` submodule.

Reply:
Okay!